examples/features/scale-from-zero/README.md

```diff
@@ -0,0 +1,102 @@
+# Test automatic scale from zero
+
+This example shows that NSEs can be created on the fly, allowing effective scaling.
+
+## Run
+
+Create test namespace:
+```bash
+NAMESPACE=($(kubectl create -f ../namespace.yaml)[0])
+NAMESPACE=${NAMESPACE:10}
+```
+
+Register namespace in `spire` server:
+```bash
+kubectl exec -n spire spire-server-0 -- \
+/opt/spire/bin/spire-server entry create \
+-spiffeID spiffe://example.org/ns/${NAMESPACE}/sa/default \
+-parentID spiffe://example.org/ns/spire/sa/spire-agent \
+-selector k8s:ns:${NAMESPACE} \
+-selector k8s:sa:default
+```
+
+Create customization file:
+```bash
+cat > kustomization.yaml <<EOF
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: ${NAMESPACE}
+
+bases:
+- ../../../apps/nse-supplier-k8s
+- ../../../apps/nsc-kernel
+
+resources:
+- supplier-role.yaml
+
+configMapGenerator:
+- name: supplier-pod-template-configmap
+  files:
+  - pod-template.yaml
```

Comment:
Can we also move this into `apps/nse-supplier-k8s/kustomization.yaml`?

Reply:
We can.
I placed the template in the example folder because I feel like it's a part of the example, and if we had a different example using the supplier, we would probably want to use another pod template.
I guess it's just a matter of preference: to have a default pod template, or to force the person, who uses supplier, always provide their own template.

Reply:
done